zlibWrapper/examples/minigzip.c

```diff
@@ -234,7 +234,7 @@ int gzwrite _Z_OF((gzFile, const void *, unsigned));
 
 int gzwrite(gzFile gz, const void *buf, unsigned len) {
     z_stream *strm;
-    unsigned char out[BUFLEN];
+    unsigned char out[BUFLEN] = 0;
```

Comment:
Did you meant `{ 0 }` ?

Reply:
God damn, I'm absent-minded...
Sorry I made so much mistakes in something that's supposed to fix stuff...

Will fix this though, thanks for noticing